dev_guide/managing_images.adoc

```diff
@@ -455,6 +456,7 @@ mountable inside of a pod. You can do this by running:
 ----
 $ oc secrets add serviceaccount/builder secrets/<pull_secret_name>
 ----
+endif::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
```

Comment:
@aweiteka You are currently leaving out the "Accessing the Internal Registry" section. Is that intended? There is an anchor reference to it later in the "Creating an Image Stream by Manually Pushing an Image" section.